src/xdocs/config_system_properties.xml

```diff
@@ -16,39 +16,14 @@
       </macro>
     </section>
 
-    <section name="Caching Support">
+    <section name="Localization Support">
       <p>
-        The property <code>checkstyle.cache.file</code>
-        specifies the name of a file that can be used to cache details
-        of files that pass Checkstyle. This can <i>significantly</i>
-        increase the speed of checkstyle on successive runs. The
-        property type is <a href="property_types.html#String">String</a>
-        and defaults to an empty string (which means no caching is done).
-      </p>
-    </section>
-
-    <section name="Localisation Support">
-      <p>
-        Checkstyle supports a mechanism for localising the output
-        messages. If your language is not supported, please consider
+        Checkstyle supports localization of the output messages.
+        Locale is taken from default java property <code>user.language</code>.
```

Comment:
Do we want it reference it can be overridden at https://checkstyle.org/config.html#Checker_Properties ?

Reply:
it might be reasonable to remove explanation of ` default java property <code>user.language</code>` it is true in all applications, and we do not do any fancy logic on it.

Reply:
I thought that it might be a useful information, but I do not insist :) 
I removed this sentence